pkg/processor/test/suite/suite.go

```diff
@@ -60,6 +61,19 @@ type TestSuite struct {
 	CleanupTemp  bool
 }
 
+// StressRequest holds information for blastHTTP function
+type StressRequest struct {
+	Duration      time.Duration
+	URL           string
+	Method        string
+	FunctionName  string
+	FunctionPath  string
+	Handler       string
+	RatePerWorker float64
+	Workers       uint64
```

Comment:
It's canonical to use `int` rather than `uint64` and `uint32` unless you need to more advanced stuff like serialization / memory mapping. In addition, can't rate be an int as well? Keep it simple if possible

Reply:
Isn't it better not to convert every time from int to uint? 
Rate can't be int...

Reply:
As far as i can see, vegeta expects rate to be uint64, workers is some type and timeout is time.Duration 

Accordingly, if you set rate / workers to int, timeout to duration and do all the conversions when you call vegeta.Attack you make the BlastHTTP interface more natural. This is what I'd expect to populate as a user of BlastHTTP. The fact that BlastHTTP uses vegeta and its types should not bleed into the API